### DISCUSSION TITLE
Crazyflie Bolt 1.1's NRF was "erased"

### DISCUSSION THREAD

**Discussion by CasperLiZ** [bitcraze/discussions#1603] [Q&A]
Hi, sorry, I use the command "nrfjprog --eraseall -f nrf52" for NRF when using Crazyflie Bolt, what should I do?
I tried using[flash_softdevice](https://github.com/bitcraze/crazyflie2-nrf-bootloader/blob/1dadfe4656930a60bbfaefa800b953551ca2d7f7/Makefile#L242)and[flash_mbs](https://github.com/bitcraze/crazyflie2-nrf-bootloader/blob/1dadfe4656930a60bbfaefa800b953551ca2d7f7/Makefile#L247C1-L247C10),but the battery voltage display was not normal. Later I found [crazyflie2-nrf-mbs](https://github.com/bitcraze/crazyflie2-nrf-mbs) , but there was no such option. I found [nrf_mbs_cfbolt11_v1.1.bin](https://github.com/bitcraze/crazyflie2-nrf-mbs/releases/download/v1.1/nrf_mbs_cfbolt11_v1.1.bin). How should I download it?

**Comment by tobbeanton** [comment]
This is a bit problematic as it is done with other tools in production. Can you please send us an emailt to support@bitcraze.io and we will try to help you out.
